F/Faiss/common.jl

```diff
@@ -11,12 +11,21 @@ apk del cmake
 
 cd faiss
 
+atomic_patch -p1 ../patches/faiss-cmake-mkl-optional.patch
 atomic_patch -p1 ../patches/faiss-mingw32-cmake.patch
 atomic_patch -p1 ../patches/faiss-mingw32-InvertedListsIOHook.patch
 atomic_patch -p1 ../patches/faiss-mingw32.patch
 
 cmake_extra_args=()
 
+if [[ "$bb_full_target" != armv6l-linux-* ]]; then
+    libblastrampoline=$(find $libdir -name libblastrampoline'*'.$dlext) # Enable Windows builds to find libblastrampoline
+    cmake_extra_args+=(
+        "-DBLAS_LIBRARIES=$libblastrampoline"
+        "-DLAPACK_LIBRARIES=$libblastrampoline"
+    )
+fi
```

Comment:
@giordano Is there a nicer way to make CMake recognize lbt as BLAS and LAPACK libraries?

Reply:
If upstream actually uses the proper CMake modules for BLAS/LAPACK, then you can just copy what SuiteSparse does and use the newest CMake and pass a single variable: https://github.com/JuliaPackaging/Yggdrasil/pull/9207/files